docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md

```diff
@@ -63,7 +63,7 @@ The samples and walkthroughs in the LINQ to DataSet documentation use the Advent
   
 1. Start Internet Explorer.  
```

Comment:
Seems like an outdated article ?
@mairaw, Does this suggestion seem to be good ?

```suggestion
1. Start your favorite browser, such as Microsoft Edge.
```